frb/tests/test_galfit.py

```diff
@@ -0,0 +1,43 @@
+# Module to test 
+# the galfit wrapper
+
+import pytest
+import os
+import shutil
+import numpy as np
+
+from astropy.io import fits
+from astropy.table import Table
+from astropy.wcs import WCS
+
+from pkg_resources import resource_filename
+
+from frb.galaxies.frbgalaxy import FRBHost
+
+from frb.galaxies import galfit as glf
+
+
+def test_platescale():
+    cutout_file = resource_filename('frb','data/analysis/GALFIT/cutout_DES_i.fits')
+    _, hdr = fits.getdata(cutout_file, header=True)
+    wcs = WCS(hdr)
+    platescale = glf.get_platescale(wcs)
+    assert np.isclose(platescale, 0.263)
+
+def test_run():
+    cutout_file = resource_filename('frb','data/analysis/GALFIT/cutout_DES_i.fits')
```

Comment:
Ah, I see those files are for testing.  Can we move them
into `frb/tests/files/` ?  Thanks @SunilSimha

Reply:
Moved.

Reply:
I see Travis is failing at `test_galfit.run`. I suspect this is because it doesn't really have access to the program and its server is returning a "command `galfit` not found" error.